### PR TITLE
DDF-2403 Fixing small issues in the STS Ldap Login docs

### PR DESCRIPTION
**Pull request by brjeter**
#### What does this PR do?
Quick docs fix for typos
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@ricklarsen @figliold @paouelle 

[DDF-2403](https://codice.atlassian.net/browse/DDF-2403)

### CODE REVIEW

File: distribution/docs/src/main/resources/_contents/_security-framework/security-sts-ldap-login-contents.adoc [hunk 1]
```diff
@@ -54,8 +54,10 @@ Configure the Security STS LDAP Claims Handler from the ${admin-console}:
 
 |===
 
-.Security STS LDAP Claims Handler Imported Services
+===== Implementation Details
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_security-framework/security-sts-ldap-login-contents.adoc]: Having these as section titles will create automatic anchors to this section and the name `Implementation Details` will be too general considering all the other sections that _could_ be named the same. 

I'm not sure we even need to include this information, especially as nothing is imported or exported. Unless we have a use case, I'd advise just deleting these everywhere.

File: distribution/docs/src/main/resources/_contents/_security-framework/security-sts-ldap-login-contents.adoc [hunk 2]
```diff
@@ -3,20 +3,20 @@
...
 This bundle is not installed by default but can be added by installing the `security-sts-ldaplogin` feature.
 
-===== Configuring the Security STS LDAP Claims Handler
+===== Configuring the Security STS LDAP Login
 
-Configure the Security STS LDAP Claims Handler from the ${admin-console}:
+Configure the Security STS LDAP Login from the ${admin-console}:
 
 . Navigate to the ${admin-console}.
 . Select *${ddf-security} Application*.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_security-framework/security-sts-ldap-login-contents.adoc]: I know this isn't part of your changes, but can you change the `*` (bold) to only be around the `${ddf-security}` and not `Application`?

**Comment by brjeter** [comment]
Comments addressed

**Comment by brjeter** [comment]
build now

**Pull request merged**